checks/sanity

```diff
@@ -56,11 +56,11 @@ check_smtcomp_status unsat --mode smtcomp ucore/test1.smt2
 check_exact_output ucore/test2.out --input_syntax smtlib2 -om ucore ucore/test2.smt2
 
 # Integer induction problems
-check_szs_status Unsatisfiable -ind int ind/int_invariant_infinite_geq3_val3.smt2
-check_szs_status Unsatisfiable -ind int ind/int_invariant_finite_a_to_b.smt2
-check_szs_status Unsatisfiable --forced_options "drc=off:ind=int:intindint=infinite:sos=theory:sstl=1:to=lpo" ind/int_power_0_all_0.smt2
-check_szs_status Unsatisfiable --forced_options "ind=int:sa=discount:sos=theory:sstl=1:to=lpo" ind/int_sum_y_geq_0.smt2
+check_szs_status Unsatisfiable --mode portfolio --schedule integer_induction --slowness 1 ind/int_invariant_infinite_geq3_val3.smt2
```

Comment:
I think I would prefer invoking the exact strategy from this portfolio that solves the problem, otherwise this might break when we change the schedules. But I don't feel particularly strongly one way or another. :-)

Reply:
I'd actually prefer to use portfolio. The proofs that Vampire produces for these problems have up to 100 steps (it's not easy to find problems that require various types of integer induction and have short proofs), which means that if something unrelated changes in the proof search, Vampire might diverge and not find the proof. See Slack: https://vampireglobal.slack.com/archives/C014NQS2UNL/p1619758673000900
If we use the portfolio mode, there should always be a couple of schedules that solve each of the problems, so the sanity check won't break that easily.